modules/friendlyshared.js

```diff
@@ -16,15 +16,17 @@
 Twinkle.shared = function friendlyshared() {
 	if( mw.config.get('wgNamespaceNumber') === 3 && Morebits.isIPAddress(mw.config.get('wgTitle')) ) {
 		var username = mw.config.get('wgTitle').split( '/' )[0].replace( /\"/, "\\\""); // only first part before any slashes
-		Twinkle.addPortletLink( function(){ Twinkle.shared.callback(username); }, "Shared IP", "friendly-shared", "Shared IP tagging" );
+		Twinkle.addPortletLink( function(){ Twinkle.shared.callback(username); }, $.i18n._('tw-sharedip-label'), "friendly-shared", 'tw-sharedip-label-hover' );
```

Comment:
Will do. I was hoping to get more comments on the bigger picture though, rather than function naming. Specifically: the "logic" of the tool is embedded into the code and no matter how much we separate the message texts (i18n) and references to them (i10n), we still will have lines of code that are EN WP logic and may not apply to another wiki. How do we want to proceed with them?

In other words, if we are fine with EN WP logic being in the code, and the only limiting issue is the name of _ function, I'll happily fix that!